cpp/tests/search/search_list_test.cpp

```diff
@@ -347,3 +347,66 @@ TYPED_TEST(TypedListContainsTestColumnNeedles, ListsOfStructs)
   auto const result   = cudf::contains(*haystack, *needles);
   CUDF_TEST_EXPECT_COLUMNS_EQUAL(expected, *result, verbosity);
 }
+
+struct ListLowerBound : public cudf::test::BaseFixture {
+};
+
+TEST_F(ListLowerBound, ListWithNulls)
+{
+  {
+    using lcw = cudf::test::lists_column_wrapper<double>;
+    using cudf::test::iterators::nulls_at;
+    lcw col1{
+      lcw{-3.45967821e+12},  // 0
+      lcw{-3.6912186e-32},   // 1
+      lcw{9.721175},         // 2
+    };
+
+    lcw val1{
```

Comment:
We don't have `col2` thus let's not use the number in var name.
```suggestion
    auto const haystack = lcw{
      lcw{-3.45967821e+12},  // 0
      lcw{-3.6912186e-32},   // 1
      lcw{9.721175},         // 2
    };

    auto const needles = lcw{
```